model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -132,10 +132,18 @@ object AddressResponseAddress {
     * @param other
     * @return
     */
-  def fromPafAddress(maxScore: Float)(other: PostcodeAddressFileAddress): AddressResponseAddress =
+  def fromPafAddress(maxScore: Float)(other: PostcodeAddressFileAddress): AddressResponseAddress = {
+
+    val poBoxNumber = if (other.poBoxNumber == "") "" else s"PO BOX ${other.poBoxNumber}"
+
+    val formattedAddress = Seq(other.departmentName, other.organizationName, other.subBuildingName,
```

Comment:
Because it will fill the string with `${}`.
And also I filter the sequence to remove empty strings